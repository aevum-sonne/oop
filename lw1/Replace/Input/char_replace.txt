Leger believed that the mechanical age had the capability of curing the chaos that occurred during World War I, and reduced the figures in his paintings to mechanical shapes for this very purpose. In Woman with a Book, Leger has reduced the shape of the woman down to simple forms, which have been placed back together like a machine.
This portrait thus gives of the impression that the woman is a mechanized version of herself. The simple, solidified forms were Leger’s attempt to use enduring images to recover from the devastation of the war. This portrait gives off the impression that the woman’s strong stance and solid figure offers comfort after the tribulations of war.
