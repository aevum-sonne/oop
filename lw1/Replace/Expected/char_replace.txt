L1g1r b1li1v1d that th1 m1chanical ag1 had th1 capability of curing th1 chaos that occurr1d during World War I, and r1duc1d th1 figur1s in his paintings to m1chanical shap1s for this v1ry purpos1. In Woman with a Book, L1g1r has r1duc1d th1 shap1 of th1 woman down to simpl1 forms, which hav1 b11n plac1d back tog1th1r lik1 a machin1.
This portrait thus giv1s of th1 impr1ssion that th1 woman is a m1chaniz1d v1rsion of h1rs1lf. Th1 simpl1, solidifi1d forms w1r1 L1g1r’s att1mpt to us1 1nduring imag1s to r1cov1r from th1 d1vastation of th1 war. This portrait giv1s off th1 impr1ssion that th1 woman’s strong stanc1 and solid figur1 off1rs comfort aft1r th1 tribulations of war.
